modules/core/src/main/java/org/apache/ignite/internal/managers/communication/ErrorMessage.java

```diff
@@ -38,10 +38,10 @@
 public class ErrorMessage implements Message {
     /** Serialized form of throwable. */
     @Order(value = 0, method = "errorBytes")
-    private byte @Nullable [] errBytes;
+    private @Nullable byte[] errBytes;
```

Comment:
Let's add `"NullableProblems"` to `@SuppressWarnings` for entire class:
`@SuppressWarnings({"AssignmentOrReturnOfFieldWithMutableType", "NullableProblems"})`